crates/storage/db/Cargo.toml

```diff
@@ -38,11 +38,14 @@ parking_lot.workspace = true
 derive_more.workspace = true
 eyre.workspace = true
 paste = "1.0"
+dashmap = "5.5.3"
+rustc-hash = "1.1.0"
 
 # arbitrary utils
 arbitrary = { workspace = true, features = ["derive"], optional = true }
 proptest = { workspace = true, optional = true }
 proptest-derive = { workspace = true, optional = true }
+strum = { workspace = true, features = ["derive"] }
```

Comment:
this is fine, because we already had `strum` in `reth-primitives` that practically almost every crate depends on